presto-plugin-toolkit/src/test/java/io/prestosql/plugin/base/security/TestFileBasedAccessControl.java

```diff
@@ -64,15 +68,23 @@ public void testTableRules()
         accessControl.checkCanSelectFromColumns(user("bob"), new SchemaTableName("bobschema", "bobtable"), ImmutableSet.of());
         accessControl.checkCanInsertIntoTable(user("bob"), new SchemaTableName("bobschema", "bobtable"));
         accessControl.checkCanDeleteFromTable(user("bob"), new SchemaTableName("bobschema", "bobtable"));
-        accessControl.checkCanSelectFromColumns(user("joe"), new SchemaTableName("bobschema", "bobtable"), ImmutableSet.of());
```

Comment:
- please use different name so no existing assertions change.
 - also I don't like such names that do not mean anything. For example instead of `joe` you could  use `user_without_access_to_schema_x`.

Reply:
sounds good

Reply:
Please restore the removed line:
```
        accessControl.checkCanSelectFromColumns(user("joe"), new SchemaTableName("bobschema", "bobtable"), ImmutableSet.of());

```